graylog2-web-interface/src/pages/RulesPage.tsx

```diff
@@ -95,6 +104,28 @@ const RulesPage = () => {
     };
   };
 
+  const onCloseMetricsConfig = () => {
+    _loadRuleMetricData(setIsMetricDataLoading, setMetricsConfig);
```

Comment:
While this works, it also hides the debug metrics button while re-fetching the information. During the first load, not having the button is slightly odd but I think this issue becomes really prominent when working with the modal (since it will load every time it is closed), and specially in a slower connection.



https://user-images.githubusercontent.com/716185/146947279-978451f1-ac6e-4ffc-afe4-e38a4cc06b78.mp4


Why do we hide the button on loading in the first place? I think we should not hide it, either display the standard "Debug metrics" while we load, or show a loading state in the button.

Reply:
Hi, thank you for testing. We were waiting for the metrics config data to show the button which also shows the debug config state. I made the change to show the disabled state when loading.